src/main/java/org/jabref/gui/WelcomeTab.java

```diff
@@ -0,0 +1,274 @@
+
+        return footerBox;
+    }
+
+    private Label createFooterLabel(String text) {
+        Label label = new Label(text);
+        label.getStyleClass().add("welcome-footer-label");
+        return label;
+    }
+
+    private HBox createIconLinksContainer() {
+        HBox container = new HBox(15);
+        container.setAlignment(Pos.CENTER);
+
+        Hyperlink onlineHelpLink = createFooterLink(Localization.lang("Online help"), StandardActions.HELP, IconTheme.JabRefIcons.HELP);
+        Hyperlink forumLink = createFooterLink(Localization.lang("Forum for support"), StandardActions.OPEN_FORUM, IconTheme.JabRefIcons.FORUM);
+        Hyperlink mastodonLink = createFooterLink(Localization.lang("Mastodon"), StandardActions.OPEN_MASTODON, IconTheme.JabRefIcons.MASTODON);
+        Hyperlink linkedInLink = createFooterLink(Localization.lang("LinkedIn"), StandardActions.OPEN_LINKEDIN, IconTheme.JabRefIcons.LINKEDIN);
+        Hyperlink donationLink = createFooterLink(Localization.lang("Donation"), StandardActions.DONATE, IconTheme.JabRefIcons.DONATE);
+
+        container.getChildren().addAll(onlineHelpLink, forumLink, mastodonLink, linkedInLink, donationLink);
+        return container;
+    }
+
+    private HBox createTextLinksContainer() {
+        HBox container = new HBox(15);
+        container.setAlignment(Pos.CENTER);
+
+        Hyperlink devVersionLink = createFooterLink(Localization.lang("Download development version"), StandardActions.OPEN_DEV_VERSION_LINK, null);
+        Hyperlink changelogLink = createFooterLink(Localization.lang("CHANGELOG"), StandardActions.OPEN_CHANGELOG, null);
+
+        container.getChildren().addAll(devVersionLink, changelogLink);
+        return container;
+    }
+
+    private Hyperlink createFooterLink(String text, StandardActions action, IconTheme.JabRefIcons icon) {
+        Hyperlink link = new Hyperlink(text);
+        link.getStyleClass().add("welcome-footer-link");
+
+        String url = switch (action) {
+            case HELP -> URLs.HELP_URL;
+            case OPEN_FORUM -> URLs.FORUM_URL;
+            case OPEN_MASTODON -> URLs.MASTODON_URL;
+            case OPEN_LINKEDIN -> URLs.LINKEDIN_URL;
+            case DONATE -> URLs.DONATE_URL;
+            case OPEN_DEV_VERSION_LINK -> URLs.DEV_VERSION_LINK_URL;
+            case OPEN_CHANGELOG -> URLs.CHANGELOG_URL;
+            default -> null;
+        };
+
+        if (url != null) {
+            link.setOnAction(e -> new OpenBrowserAction(url, dialogService, preferences.getExternalApplicationsPreferences()).execute());
+        }
+
+        if (icon != null) {
+            link.setGraphic(icon.getGraphicNode());
+        }
+
+        return link;
+    }
+
+    private HBox createVersionContainer() {
+        HBox container = new HBox(15);
+        container.setAlignment(Pos.CENTER);
+
+        Label versionLabel = new Label(Localization.lang("Current JabRef version") + ": " + buildInfo.version);
```

Comment:
Sorry again

GOOD: Localization.lang("Current JabRef version: %0",  buildInfo.version);

Use parameters. Reason: even the `:` can be localized